### PR TITLE
fix: digest options as array

### PR DESCRIPTION
**Pull request by rebelchris**
We can review it, but can only merge once FE changes are done :) 
The idea is that we now have 2 already and third option coming so we need to get all for set user.

### CODE REVIEW

File: src/schema/users.ts [hunk 1]
```diff
@@ -1104,16 +1117,14 @@ export const resolvers: IResolvers<any, Context> = {
...
       ctx: Context,
-    ): Promise<GQLPersonalizedDigest> => {
+    ): Promise<GQLPersonalizedDigest[]> => {
       const personalizedDigest = await ctx
         .getRepository(UserPersonalizedDigest)
-        .findOneBy({ userId: ctx.userId, type });
+        .findBy({ userId: ctx.userId });
 
-      if (!personalizedDigest) {
+      if (!personalizedDigest || personalizedDigest.length === 0) {
```
Comment (capJavert) [inline src/schema/users.ts:1127]: its always gonna be array so latter is enough

**Review by capJavert — APPROVED** [review]

File: src/schema/users.ts (hunk 1 above)
Comment (omBratteng) [reply src/schema/users.ts:1127]:
```suggestion
      if (personalizedDigest?.length === 0) {
```
maybe?

Comment (capJavert) [reply src/schema/users.ts:1127]: it will never be undefined, `findBy` returns array

**Comment by rebelchris** [comment]
@capJavert Also had to add sendType flag to mutation as we now allow it to be passed from FE.

**Review by capJavert — APPROVED** [review]

**Pull request merged**